core/src/main/java/org/commonjava/indy/core/ctl/NfcController.java

```diff
@@ -55,7 +56,7 @@
 {
 
     @Inject
-    protected AbstractNotFoundCache cache;
+    protected NotFoundCache cache;
```

Comment:
I'm thinking if we need a new interface like IndyNotFoundCache or something to include the missing methods in AbstractNFC. I always think that these class casting is ugly and misleading....

Reply:
Yeah they looks ugly. Maybe we should abandon those addtional methods. They are mainly for pagination and we find they are almost useless. The cassandra impl does not implement them. We can get back to it later if needed.